metrotranscom/doorway
# feat: add bathroom filter

# Pull Request Template

## Issue Overview


## Description

This PR adds a filter on the number of bathrooms in any of the units on a listing.  It supports numeric comparison operations, so it can be used to respond to a request for "listings with units that have greater than or equal to 2 bathrooms".

## How Can This Be Tested/Reviewed?

There is an e2e test under `listings-combined.e2e-spec.ts`, same as with the other filters.  Just run `yarn test:e2e:local` or `yarn test:e2e:local -t combined` (faster) to verify test results.  You can also run `yarn test` to run unit tests if you want to be thorough.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
